docs/Parameters/PeakMap.rst

```diff
@@ -3,7 +3,7 @@ Peak Map
 
 Peak Maps can be plotted using kind = peakmap. Commonly in this plot, mass-to-charge is on the x-axis, retention time is on the y-axis and intensity is on the z-axis (or represented by color). The x and y axis can be changed based on use case, for example y can also be ion mobility. Using `plot_3d=True` enables 3D plotting. Currently 3D plotting only supported for `ms_matplotlib` and `ms_plotly` backends.
 
-
+Note: y_kind / x_kind is only relevant for this kind and if add_marginals is set to True.
```

Comment:
Sorry nitpicky comment. Reads strangely with many "kinds" 

Please change to "Note: y_kind / x_kind is only relevant if add_marginals is set to True." 

Also please change other instances of this sentence below.